src/data/path_spawn_data.c

```diff
@@ -2,13 +2,29 @@
 #include "courses/all_course_data.h"
 #include "ending/ceremony_data.inc.h"
 
-UnkCommonTextureStruct0 *D_800DC720[] = {
-	D_0D008F28, D_0D008F80, D_0D008FB8, D_0D009058,
-	D_0D0090B8, D_0D0090F8, D_0D009158, D_0D009188,
-	D_0D0091E8, D_0D009210, D_0D009238, D_0D009260,
-	D_0D009280, D_0D0092C8, D_0D009310, D_0D008F18,
-	D_0D008F18, D_0D008F18, D_0D0093C0, D_0D008F18,
-	D_0D008F18, NULL,
+KartAIBehaviour *gKartAIBehaviourLUT[NUM_COURSES+1] = {
```

Comment:
This is fine, because my PR will change this array completely.

For future reference, don't hard-coded the array size in `[]` because that means the array will not shift.

Albeit, perhaps a good way to get a warning that you forgot to update X array. So now that I think about it, it *could* be useful. But sometimes it's better to just let the array shift if things are added.

Reply:
hmm ok I think a more thought about how unfix fix array are needed but if torch will generate this code it's fine

Reply:
In C you don't need to put anything in `[]`. C calculates the array size automatically and passes it onto other files so long as you have declared it in the header.

The only time we need to hard-coded the array size is when we can't link 2 files together properly (course textures and course displaylists or whatever file it is for example.)

Reply:
ok ok